setup.py

```diff
@@ -14,15 +14,15 @@
     version=version,
     author="Richard Hull",
     author_email="richard.hull@destructuring-bind.org",
-    description="A library to drive a MAX7219 LED serializer using hardware spidev",
+    description="A library to drive a MAX7219 LED serializer (using SPI) and WS2812 NeoPixels (using DMA)",
     long_description="\n\n".join([README, CONTRIB, CHANGES]),
     license="MIT",
-    keywords=["raspberry pi", "rpi", "led", "max7219", "matrix", "seven segment", "7 segment"],
+    keywords=["raspberry pi", "rpi", "led", "max7219", "matrix", "seven segment", "7 segment", "neopixel", "ws2812", "ws281x"],
     url="https://github.com/rm-hull/luma.led_matrix",
     download_url="https://github.com/rm-hull/luma.led_matrix/tarball/" + version,
     namespace_packages=["luma"],
     packages=["luma.led_matrix"],
-    install_requires=["luma.core>=0.1.11"],
+    install_requires=["luma.core>=0.1.11", "ws2812"],
```

Comment:
I looked into the extra_requires and it seems unnecessary complex, so +1 for this.